docs/run-sparkbench.md

```diff
@@ -88,3 +88,59 @@ hibench.yarn.executor.num   |   Spark executor number in Yarn mode
 hibench.yarn.executor.cores  |  Spark executor cores in Yarn mode
 spark.executor.memory  | Spark executor memory
 spark.driver.memory    | Spark driver memory
+
+
+### 8. Run xgboost workload ###
+
+Hibench xgboost benchmark depends on the xgboost libraries to build and run. The libs are ```xgboost4j_<scala version>-<xgboost version>.jar``` and ```xgboost4j-spark_<scala version>-<xgboost version>.jar```.<br>
+The relevant configurations are in ```./sparkbench/ml/pom.xml```
+```
+    <dependency>
+      <groupId>ml.dmlc</groupId>
+      <artifactId>xgboost4j_${scala.binary.version}</artifactId>
+      <version>1.1.0</version>
+    </dependency>
+    <dependency>
+      <groupId>ml.dmlc</groupId>
+      <artifactId>xgboost4j-spark_${scala.binary.version}</artifactId>
+      <version>1.1.0</version>
+    </dependency>
+```
+and ```./pom.xml```
+```
+    <repository>
+      <id>xgboostrepo</id>
+      <name>XGBoost Maven Repo</name>
+      <url>https://s3-us-west-2.amazonaws.com/xgboost-maven-repo/release</url>
+      <releases>
+        <enabled>true</enabled>
+      </releases>
+      <snapshots>
+        <enabled>false</enabled>
+      </snapshots>
+    </repository>
+```
+
+#### 8.a latest xgboost release (default) ####
+
```

Comment:
don't need to use 8.a, 8.b., need to use correct captial cases for titles.

Reply:
I think you don't need to write this since it's already written in the above section 4. Run a workload